### PR TITLE
Import project

### PR DESCRIPTION
**Pull request by zoltanszocs**
## Checklist

- [ ] Keep pull requests small so they can be easily reviewed.
- [ ] Update the documentation, please follow the [numpy style guide](https://numpydoc.readthedocs.io/en/latest/format.html).
- [ ] Update tests.
- [ ] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `feature`, `ui`, `change`, `documentation`, `breaking`, `ci`
      as they show up in the changelog.
- [ ] Link this PR to related issues by using [closing keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).
- [ ] Add a milestone to the PR for the intended release.
- [ ] Request a review from another developer.

### CODE REVIEW

File: activity_browser/actions/project/project_remote_import.py [hunk 1]
```diff
@@ -93,27 +103,117 @@ def __init__(self):
...
+        self.import_button.clicked.connect(self._import_project)
         layout.addLayout(import_button_layout)
 
         self.setLayout(layout)
 
+    def _populate_table(self):
+        url_path = self.remote_url_path.text()
+        if url_path[-1] != "/":
+            url_path += "/"
+        self.table.populate(url_path + self.remote_catalogue.text())
```
Comment (cmutel) [inline activity_browser/actions/project/project_remote_import.py]: Let's use `urllib.parse.urljoin` here instead.

**Review by cmutel — CHANGES_REQUESTED** [review]
Nice!
Screenshot 2024-08-12 at 21 05 17
- [ ] Can we add some more space here? It feels quite cramped.
- [ ] Can we please give some indication on *why* the "create project" button is disactivated, i.e. warning text which states that the project name already exists
- [ ] Can we add another checkbox to switch to the project after import? The code to run would be `bw2data.projects.set_current(<new_project_label>)`

File: activity_browser/actions/project/project_remote_import.py (hunk 1 above)
Comment (zoltanszocs) [reply activity_browser/actions/project/project_remote_import.py]: Done

**Comment by zoltanszocs** [comment]
- Added a bit more space between components - I was not entirely sure where you wanted more space, increasing the global spacing in the layout seemed strange. Let me know how this looks.
- Added status label at the bottom, displaying what is the next necessary step: load catalogue, select project, project already exists
- Added handling for the case when there is no valid catalogue loaded
- Added busy indication for the catalogue loading and project importing
- Added new activate checkbox and functionality

**Pull request merged**